src/wkwebview/mod.rs

```diff
@@ -226,11 +228,17 @@ impl InnerWebView {
         let function = Box::into_raw(Box::new(function));
         protocol_ptrs.push(function);
 
-        let ivar = (*handler).class().instance_variable("function").unwrap();
+        let ivar = (*handler)
+          .class()
+          .instance_variable(CStr::from_bytes_with_nul(b"function\0").unwrap())
```

Comment:
why this in favor of `c""` ? I saw a mention of that in your changelog and used that instead. (i never really used cstr before so idk)

Reply:
`c""` is only available on newer Rust versions (can't remember the version right now, but it's something like 1.80)

Reply:
(I'm not sure what your MSRV is, so I remained conservative)

Reply:
ohhh, okay. Yeah, for use in tauri we're stuck on 1.77 for a while (win7)